docs/framework/docker/aspnetmvc.md

```diff
@@ -50,43 +45,39 @@ The finished application is located in the
 
 ## Prerequisites
 
-At a minimum, your development machine must be running
-[Windows 10 Anniversary Update](https://www.microsoft.com/en-us/software-download/windows10/)
-or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server). 
+The development machine must be running
+[Windows 10 Anniversary Update](https://www.microsoft.com/en-us/software-download/windows10/) (or higher)
+or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server) (or higher). 
 
-Before starting, you need to install [Docker for Windows](https://docs.docker.com/docker-for-windows/),
-version 1.12 Beta 26, or newer. Windows Container support is only available
-in the Beta channel at this time.
+[Docker for Windows](https://docs.docker.com/docker-for-windows/)
```

Comment:
@mairaw  can we link to that so we don't need to update this doc every couple of weeks? I don't think we even need that. The link https://docs.docker.com/docker-for-windows has at the top the stable and the beta channel. The Stable channel works. The Beta changes frequently. Folks who go with the Beta can follow the instructions on that page, no need to duplicate them here.

@spboyer  can you comment?